Wrong dosemu fonts

with 

`$_X_font = "vga11x19"`

dosemu gives this:

```
You do not have the DOSEMU vga11x19 font installed and are running
remote X. You need to install the vga11x19 font on your _local_ Xserver.
Look at the readme for details. For now we start with the bitmapped
built-in font instead, which may be slower.

```
Here is the output of  /usr/share/dosemu/Xfonts

```
gilang@gilang-desktop:~/master/dosemu2$ ls /usr/share/dosemu/Xfonts -l
total 76
-rw-r--r-- 1 root root  784 Apr 17 20:23 fonts.alias
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga10x20-cp850.pcf.gz
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga10x20-cp866.pcf.gz
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga10x24.pcf.gz
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga11x19.pcf.gz
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga12x30.pcf.gz
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga8x14-cp850.pcf.gz
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga8x19.pcf.gz
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga-cp866.pcf.gz
-rw-r--r-- 1 root root 5537 Apr 17 20:23 vga.pcf.gz

```

The file sizes are different with dosemu1's, here is dosemu1's:

```
bambang@bambang-notebook:~$ ls -l /usr/local/share/dosemu/Xfonts
total 364
-rw-r--r-- 1 root root 307200 Apr 17 15:05 -z
-rw-r--r-- 1 root root    637 Oct 26  2012 fonts.alias
-rw-r--r-- 1 root root    518 Oct 26  2012 fonts.dir
-rw-r--r-- 1 root root   5011 Oct 26  2012 vga-cp866.pcf.gz
-rw-r--r-- 1 root root   4844 Oct 26  2012 vga.pcf.gz
-rw-r--r-- 1 root root   5451 Oct 26  2012 vga10x20-cp866.pcf.gz
-rw-r--r-- 1 root root   5702 Oct 26  2012 vga10x24.pcf.gz
-rw-r--r-- 1 root root   5838 Oct 26  2012 vga11x19.pcf.gz
-rw-r--r-- 1 root root   6271 Oct 26  2012 vga12x30.pcf.gz
-rw-r--r-- 1 root root   5323 Oct 26  2012 vga8x19.pcf.gz


```

By copying dosemu1's fonts directory, the problem is solved.

This was unnoticed before because I have a copy of  vga11x19.pcf.gz from previous version in other font path.

The debian build seems to have broken for all versions and archs
~~~
echo "-> Installing the X PC fonts..."
-> Installing the X PC fonts...
for i in ../etc/*.pcf.gz; do \
    install -m 0644 $i /<<PKGBUILDDIR>>/debian/dosemu2/usr/share/dosemu/Xfonts; \
done
install -m 0644 ../etc/fonts.dir /<<PKGBUILDDIR>>/debian/dosemu2/usr/share/dosemu/Xfonts
install: cannot stat '../etc/fonts.dir': No such file or directory
../src/arch/linux/Makefile.main:153: recipe for target 'install' failed
make[2]: *** [install] Error 1
make[2]: Leaving directory '/<<PKGBUILDDIR>>/src'
Makefile:26: recipe for target 'install' failed
make[1]: *** [install] Error 2
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
dh_auto_install: make -j4 install DESTDIR=/<<PKGBUILDDIR>>/debian/dosemu2 AM_UPDATE_INFO_DIR=no returned exit code 2
debian/rules:3: recipe for target 'binary-arch' failed
make: *** [binary-arch] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary-arch gave error exit status 2
~~~

Nope, sorry all versions and arch still failing
~~~
echo "-> Installing the X PC fonts..."
-> Installing the X PC fonts...
for i in ../etc/*.pcf.gz; do \
    install -m 0644 $i /<<PKGBUILDDIR>>/debian/dosemu2/usr/share/dosemu/Xfonts; \
done
/usr/bin/install -c -m 0644 ../etc/dosemu.alias /<<PKGBUILDDIR>>/debian/dosemu2/usr/share/dosemu/Xfonts/fonts.alias; \
cd /<<PKGBUILDDIR>>/debian/dosemu2/usr/share/dosemu/Xfonts
mkfontdir
ln -sf /<<PKGBUILDDIR>>/debian/dosemu2/usr/share/dosemu/Xfonts /etc/X11/fontpath.d/dosemu2:unscaled:pri=20
ln: failed to create symbolic link '/etc/X11/fontpath.d/dosemu2:unscaled:pri=20': No such file or directory
../src/arch/linux/Makefile.main:153: recipe for target 'install' failed
make[2]: *** [install] Error 1
make[2]: Leaving directory '/<<PKGBUILDDIR>>/src'
Makefile:26: recipe for target 'install' failed
make[1]: *** [install] Error 2
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
dh_auto_install: make -j4 install DESTDIR=/<<PKGBUILDDIR>>/debian/dosemu2 AM_UPDATE_INFO_DIR=no returned exit code 2
debian/rules:3: recipe for target 'binary-arch' failed
make: *** [binary-arch] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary-arch gave error exit status 2
~~~

BTW what's the rational for putting dosemu fonts in their own directory? On my Lubuntu 16.04 system, no other package has done so.
~~~
ajb@polly:~$ find /usr/share/fonts/X11 -type d
/usr/share/fonts/X11
/usr/share/fonts/X11/Type1
/usr/share/fonts/X11/util
/usr/share/fonts/X11/encodings
/usr/share/fonts/X11/encodings/large
/usr/share/fonts/X11/misc
~~~

~~~
ajb@polly:~$ xset q | egrep '(Font|fonts)' 
Font Path:
  /usr/share/fonts/X11/misc,/usr/share/fonts/X11/Type1,built-ins
~~~

and I don't even have an /etc/X11/fontpath.d

>So as you can see, most symlinks there are
owned by the packages, which means they
are not generated on install time. 

But from the error earlier the link is attempted to be made at package build time into below the real`/etc` directory, this is on the build farm logs.
~~~
>ln -sf /<<PKGBUILDDIR>>/debian/dosemu2/usr/share/dosemu/Xfonts /etc/X11/fontpath.d/dosemu2:unscaled:pri=20
ln: failed to create symbolic link '/etc/X11/fontpath.d/dosemu2:unscaled:pri=20': No such file or directory
../src/arch/linux/Makefile.main:153: recipe for target 'install' failed
make[2]: *** [install] Error 1
~~~
I know that you now test for `/etc/X11/fontpath.d` existence, but surely that can't be right?

The thing is with the right installation path for fonts in the package, the Debian package helper sorts out the font installation and removal for you. e.g. it generates the post.install script magically. Here's the one from the deb package generated with just my one line change
```sh
#!/bin/sh
set -e
# Automatically added by dh_installxfonts
if which update-fonts-dir >/dev/null 2>&1; then
	update-fonts-dir --x11r7-layout misc
fi
# End automatically added section
```

--- a/debian/rules
+++ b/debian/rules
@@ -6,4 +6,5 @@ override_dh_auto_configure:
 	./autogen.sh
 	dh_auto_configure -O--parallel -- \
 		--with-plugindir=/usr/lib/dosemu \
+		--with-x11fontdir=/usr/share/fonts/X11/misc \
 		--enable-plugin=vde

